app/assets/stylesheets/style.css.scss

```diff
@@ -181,6 +184,10 @@ a {
   }
 }
 
+.title {
+  padding-top: 100px;
```

Comment:
Could you instead add this padding to `#main-content`? Adding it to`.title` only seems unfit